itftp/itftp.go

```diff
@@ -87,17 +87,17 @@ func (t Handler) HandleRead(filename string, rf io.ReaderFrom) error {
 	content, ok := binary.Files[filepath.Base(shortfile)]
 	if !ok {
 		err := fmt.Errorf("file unknown: %w", os.ErrNotExist)
-		l.Error(err, "file unknown")
+		log.Error(err, "file unknown")
```

Comment:
I kind of like the HTTP server code's error better ("requested file not found").

Also, does this end up producing a log message like "file unknown - file unknown: file does not exist"? Since we are already importing `os`, I guess we could just return `os.ErrNotExist` and log that?

Maybe it's worth thinking about logging the requested file path as well? I believe otel gets that information, so being consistent comes to mind. I know you moved away from that in the HTTP server code... So, if we end up doing it here, we should log it in the HTTP server code as well.

Reply:
I'll add the filename to the error. the `log.Error` line will already include the filename because of [here](https://github.com/tinkerbell/ipxedust/blob/f5c5062b5b53d6b65d635251dc8a2e8827a48336/itftp/itftp.go#L57) and [here](https://github.com/tinkerbell/ipxedust/blob/f5c5062b5b53d6b65d635251dc8a2e8827a48336/itftp/itftp.go#L67).